README.md

```diff
@@ -78,14 +78,14 @@ The following attributes are available for `viam:camera:filtered-camera` bases:
 | ---- | ------ | ------------ | ----------- |
 | `camera` | string | **Required** | The name of the camera to filter images for. |
 | `vision_services` | list | **Required** | A list of 1 or more vision services used for image classifications or detections. |
-| `vision` | string | **Required** | \*\***DEPRECATED**\*\* The vision service used for image classifications or detections. |
-| `window_seconds_before` | float64 | Optional | The size of the time window (in seconds) before the condition is met, during which images are buffered. This allows you to see the photos taken in the specified number of seconds preceding the condition being met. |
-| `window_seconds_after` | float64 | Optional | The size of the time window (in seconds) after the condition is met, during which images are buffered. This allows you to see the photos taken in the specified number of seconds after the condition being met. |
-| `window_seconds` | float64 | Optional | The size of the time window (in seconds) during which images are buffered. When a condition is met, a confidence score for a detection/classification exceeds the required confidence score, the buffered images are stored, allowing us to see the photos taken in the specified number of seconds preceding and after the condition being met. If this value is set, the 'window_seconds_before' and 'window_seconds_after' variable must both be set to 0.0. |
+| `window_seconds_before` | float64 | **Required** | The size of the time window (in seconds) before the condition is met, during which images are buffered. This allows you to see the photos taken in the specified number of seconds preceding the condition being met. |
+| `window_seconds_after` | float64 |  **Required** | The size of the time window (in seconds) after the condition is met, during which images are buffered. This allows you to see the photos taken in the specified number of seconds after the condition being met. |
```

Comment:
why are these now required?

Reply:
ah I see now that `window_seconds` is being marked as deprecated in documentation? Is this related to the new Camera API with Images()?